static/app/views/settings/project/projectUserFeedback.tsx

```diff
@@ -25,7 +25,7 @@ class ProjectUserFeedbackSettings extends AsyncView<Props> {
     window.sentryEmbedCallback = function (embed) {
       // Mock the embed's submit xhr to always be successful
       // NOTE: this will not have errors if the form is empty
-      embed.submit = function (_body) {
+      embed.submit = function (_body: any) {
```

Comment:
just don't even expand the parameter then if it's unused?
```suggestion
      embed.submit = function () {
```